packages/fetchai/protocols/register/README.md

```diff
@@ -0,0 +1,39 @@
+# Register Protocol
+
+## Description
+
+This is a protocol for communication between two AEAs for registration.
+
+## Specification
+
+```yaml
+---
+name: register
+author: fetchai
+version: 0.1.0
+description: A protocol for communication between two AEAs for registration.
+license: Apache-2.0
+aea_version: '>=0.7.0, <0.8.0'
+speech_acts:
+  register:
+    info: pt:dict[pt:str, pt:str]
+  success:
+    info: pt:dict[pt:str, pt:str]
+  error:
+    error_code: pt:int
+    error_msg: pt:str
+    info: pt:dict[pt:str, pt:str]
+...
+---
+initiation: [register]
+reply:
+  register: [success, error]
+  success: []
+  error: []
+termination: [success, error]
+roles: {agent, agent}
```

Comment:
When the roles are the same, you only need to insert one:

```
roles: {agent}
```

Reply:
Yes correct, thanks